packages/animations/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.0.3
+* Fixes warnings for non-nullable bindings.
```

Comment:
Let's change all of these to "Updates for non-nullable bindings." since for the packages that did get published it's not actually fixing warnings. That's a more catch-all description that would cover the actual warning fix cases, and the cleanup of the workaround.